packages/streamid/src/event-id.ts

```diff
@@ -65,19 +82,57 @@ export class EventID {
     this._event = event
   }
 
+  static createRandom(
+    networkID: number | string,
+    networkIDOffset: number,
+    optionalParams: CreateRandomOptionalParams = {}
+  ): EventID {
+    const {
+      separatorKey = 'model',
+      separatorValue = 'kh4q0ozorrgaq2mezktnrmdwleo1d',
+      controller = 'did:key:z6MkgSV3tAuw7gUWqKCUY7ae6uWNxqYgdwPhUJbJhF9EFXm9',
+    } = optionalParams
+
+    const init = randomCID()
+    const eventHeight = Math.floor(Math.random() * 1000000)
+    const event = randomCID()
+
+    return this.create(
+      networkID,
+      networkIDOffset,
+      separatorKey,
+      separatorValue,
+      controller,
+      init,
+      eventHeight,
+      event
+    )
+  }
+
   /**
    * Create a new EventID.
    */
   static create(
```

Comment:
I was wondering if we cared about breaking changes but I'm pretty sure the answer is no, this is unused. Along those lines, do we want to export `EventID` from the index.js?

Reply:
yea this is unused (not even exported HAHA) i'll export it tho so it can be used soon